frontend/src/pages/Room/index.js

```diff
@@ -1,14 +1,37 @@
-import Button from "@mui/material/Button";
-import Grid from "@mui/material/Grid";
-import Typography from "@mui/material/Typography";
+import { useEffect } from 'react';
 
-import Editor from "react-monaco-editor";
+import { useParams } from 'react-router-dom';
 
-import ChatBlock from "../../components/ui/chat/ChatBlock";
-import QuestionBox from "../../components/ui/question/QuestionBox";
-import styles from "./Room.module.css";
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+
+import Editor from 'react-monaco-editor';
+
+import io from 'socket.io-client';
+
+import ChatBlock from '../../components/ui/chat/ChatBlock';
+import QuestionBox from '../../components/ui/question/QuestionBox';
+import styles from './Room.module.css';
+
+const communicationService = 'http://localhost:8080';
+const socket = io(communicationService);
 
 function RoomPage() {
+  // Todo: Use room id prop/state instead in the future
+  let { roomId } = useParams();
```

Comment:
Haha no reason. I used `let` because I tot changing the URL while testing might change the `roomId` hence need to be dynamic. But I forgot that refreshing a page will reassign the value so doesn't matter here. Switched back to `const` now. Thanks for pointing out!